app/models/order_notes_updater.rb

```diff
@@ -0,0 +1,46 @@
+class OrderNotesUpdater
+  attr_reader :order, :params, :notes
+
+  def initialize(order, params)
+    @order = order
+    @params = params
+  end
+
+  def update
+    return unless new_order_notes_present?
+    @notes = params[:order][:order_notes][:text]
+    # update_exsisting
+    # zero_out
+    add_new_details
```

Comment:
If the commented methods aren't going to be used right now, I would request they be deleted.
Also, should this be `add_new_notes`?